src/OpenTelemetry.Instrumentation.Http/MeterProviderBuilderExtensions.cs

```diff
@@ -34,12 +34,26 @@ public static class MeterProviderBuilderExtensions
     /// <returns>The instance of <see cref="MeterProviderBuilder"/> to chain the calls.</returns>
     public static MeterProviderBuilder AddHttpClientInstrumentation(
         this MeterProviderBuilder builder)
+    {
+        return AddHttpClientInstrumentation(builder, name: null);
+    }
+
+    /// <summary>
+    /// Enables HttpClient instrumentation.
+    /// </summary>
+    /// <param name="builder"><see cref="MeterProviderBuilder"/> being configured.</param>
+    /// <param name="name">Name which is used when retrieving options.</param>
+    /// <returns>The instance of <see cref="MeterProviderBuilder"/> to chain the calls.</returns>
+    public static MeterProviderBuilder AddHttpClientInstrumentation(
```

Comment:
Hi, you're getting some public API warnings here.
It looks like you only updated the net462 files.
You'll need to update the files for the other frameworks as well (net6.0 and netstandard2.0).

The tooling here is not obvious :)
You can either do this manually, or remove your changes and let Intellisense apply the changes for the full project.